src/ducks/balance/Balance.jsx

```diff
@@ -286,29 +290,76 @@ class Balance extends React.Component {
     this.props.router.push('/transactions')
   }
 
-  getBalanceHistory() {
-    const balanceHistories = getBalanceHistories(
-      historyData['io.cozy.bank.accounts'],
-      historyData['io.cozy.bank.operations']
-    )
+  getBalanceHistory(accounts, transactions) {
+    const balanceHistories = getBalanceHistories(accounts, transactions)
     const balanceHistory = sortBalanceHistoryByDate(balanceHistories.all)
 
     return balanceHistory
   }
 
-  getChartData() {
-    const history = this.getBalanceHistory()
+  getChartData(balanceHistory) {
     const WINDOW_SIZE = 15
 
-    const balancesSma = sma(history.map(h => h.y), WINDOW_SIZE, n => n)
+    const balancesSma = sma(balanceHistory.map(h => h.y), WINDOW_SIZE, n => n)
```

Comment:
I thought we were not doing windowing anymore ?

Reply:
Yep, I removed it in a later commit.